src/gui/sdlmain.cpp

```diff
@@ -1040,9 +1024,9 @@ static void setup_presentation_mode(FrameMode &previous_mode)
 	VGA_SetHostRate(host_rate);
 	const auto dos_rate = VGA_GetPreferredRate();
 
-	// Update the VFR duplicate countdown based on the DOS rate to produce a
-	// fixed lower-bound dupe refresh rate.
-	set_vfr_dupe_countdown_from_rate(dos_rate);
+	// Calculate the maximum number of duplicate frames before presenting
+	constexpr int8_t min_rate = 10;
```

Comment:
auto?

Reply:
`auto` will make it an `int` which (just as a type size) is too big to perfectly divide a `float`, so this will generate a precision loss warning (depending on flags). I've made it an `uint16_t`, which more accurately spans the allowable Hz range (0 Hz to 1000 Hz).

Reply:
Uh... that's a bit of misguided warning but whatever. You almost always lose precision with floats.

Reply:
(late follow up); I think the reason for the precision warnings when using `int` with `float` is that `float`'s 23 mantissa bits can't perfectly represent `int32_t` and greater types (well, specifically values > 23 bits in size, because then it needs to use the exponent, so loses some precision).

Where as for `int16_t` and smaller, they can always losslessly fit into a `float`.